firmware/cmake/stmlib.cmake

```diff
@@ -54,15 +54,18 @@ message("  🔃 Registered stm32f412rx_cube_library() function")
 # HAL_CONF_DIR: src/cubemx/Inc
 # HAL_SRCS: the ones that we want, stripped prefixes
 # SYSCALLS: most of the functions defined inside are weak references, only used to make sure it builds without error.
+# USB_ENABLED: flags if usb middleware should be included.
 function(stm32f412rx_cube_library
         HAL_LIB_NAME
         HAL_CONF_DIR
         HAL_SRCS
         SYSCALLS
         IOC_CHECKSUM
+        USB_ENABLED
```

Comment:
they all gotta be enabled no? cause chimera everywhere

Reply:
I want to support boards that don't have USB (currently thats quadruna, but future dev boards maybe? I guess on quintuna this is true but down the line it probably isnt...)

Reply:
just give them usb it's like very short on the compile times?